tests/test_visualize.py

```diff
@@ -308,3 +309,33 @@ def test_that_invalid_channel_values_raise_exception():
 @pytest.mark.parametrize("color", ["salmon", [12, 24, 48], [0, 0, 255, 40]])
 def test_that_color_is_always_returned_as_rgba_array(color):
     assert _get_color(color, 1.0).shape == (4,)
+
+
+# Test mark_boundaries()
+def test_that_invalid_mask_raises_exception():
+    with pytest.raises(ValueError):
+        mark_boundaries(image=np.zeros((10, 10)), segment_mask=np.zeros((10, 10, 10)))
+
+    with pytest.raises(ValueError):
+        mark_boundaries(image=np.zeros((10, 11)), segment_mask=np.zeros((10, 10)))
+
+
+def test_that_vaild_mask_is_accepted():
```

Comment:
````suggestion
def test_that_mark_boundaries_works_for_monochrome_images():
````

und dann noch separat
````python
def test_that_mark_boundaries_works_for_rgb_images():
````

Das lohnt sich denke ich, das in zwei Tests zu spalten, sind ja verschiedene Dinge, von denen eins funktionieren kann, das andere aber nicht